packages/mobile/src/services/notification/notification.service.ts

```diff
@@ -0,0 +1,39 @@
+import Toast from 'react-native-toast-message';
+import { NotificationType } from '~/common/enums/enums';
+import { ShowNotificationParams } from '~/common/types/types';
+
+class Notification {
+  public [NotificationType.ERROR](message: string): void {
```

Comment:
I'd suggest still add title prop, but make it optional with default value of `'Error'`. The same applies for other 2 methods